components/ILIAS/Administration/classes/class.ilAdministrationGUI.php

```diff
@@ -61,47 +62,47 @@
 */
 class ilAdministrationGUI implements ilCtrlBaseClassInterface
 {
-    protected ilObjectDefinition $objDefinition;
-    protected ilHelpGUI $help;
-    protected ilDBInterface $db;
-    public ilLanguage $lng;
-    public ilGlobalTemplateInterface $tpl;
-    public ilTree $tree;
-    public ilRbacSystem $rbacsystem;
-    public int $cur_ref_id;
-    public string $cmd;
-    public ilCtrl $ctrl;
-    protected string $admin_mode = "";
-    protected bool $creation_mode = false;
-    protected int $requested_obj_id = 0;
-    protected AdminGUIRequest $request;
-    protected ilObjectGUI $gui_obj;
+    private const array COMMANDS = ['forward', 'jump', 'jumpToPluginSlot', 'showTree'];
+
+    private readonly ilObjectDefinition $obj_definition;
+    private readonly ilHelpGUI $help;
+    private readonly ilLanguage $lng;
+    private readonly ilGlobalTemplateInterface $tpl;
+    private readonly ilTree $tree;
+    private readonly ilAccessHandler $access;
+    private ilRbacReview $rbac_review;
+    private ilObjUser $user;
```

Comment:
AFAIK they can be made `readonly` too.

```suggestion
    private readonly ilRbacReview $rbac_review;
    private readonly ilObjUser $user;
```